test/libsolidity/syntaxTests/events/event_selector_syntax.sol

```diff
@@ -2,19 +2,27 @@ library L {
     event E(bytes32, bool, bytes indexed);
 }
 
+interface I {
+    event E(bytes32, bool, bytes indexed);
+}
+
 contract B {
     event E(bytes32, bool, bytes indexed);
 }
 
 contract C is B {
     bytes32 public librarySelector = L.E.selector;
+    bytes32 public interfaceSelector = I.E.selector;
     bytes32 inheritedSelector = E.selector;
 
     function f() public {
         assert(librarySelector == L.E.selector);
+        assert(interfaceSelector == I.E.selector);
         assert(E.selector == B.E.selector);
 
         emit E(E.selector, true, "123");
+        emit I.E((B.E.selector), true, "123");
```

Comment:
This PR also solves #13928, but to prove it we should also have a non-interface case here. `B` does not count because we inherit from it.

Reply:
Like the line below (+ non-inherited `contract A`)?

Reply:
Yes. Same in the selector test.